clim-lisp.asd

```diff
@@ -4,12 +4,19 @@
   :depends-on (#:alexandria
                #:trivial-gray-streams
                #:closer-mop
-               #+(or) #:log4cl)
+               #:log4cl)
   :components (;; First possible patches
                (:file "patch")
                (:module "Lisp-Dep"
                         :components
-                        (#+(or excl clisp)
-                           (:file   #+excl      "fix-acl"
-                                    #+clisp     "fix-clisp")))
+                        (#+(or cmu scl excl sbcl openmcl lispworks clisp ecl)
+                           (:file   #+armedbear "fix-abcl"
+                                    #+cmu       "fix-cmu"
+                                    #+scl       "fix-scl"
+                                    #+excl      "fix-acl"
+                                    #+sbcl      "fix-sbcl"
+                                    #+openmcl   "fix-openmcl"
+                                    #+lispworks "fix-lispworks"
+                                    #+clisp     "fix-clisp"
+                                    #+ecl       "fix-ecl")))
```

Comment:
this seems to be based on old mcclim version, we do not have most of these files anymore